packages/connectivity/connectivity/pubspec.yaml

```diff
@@ -5,7 +5,7 @@ homepage: https://github.com/flutter/plugins/tree/master/packages/connectivity/c
 # 0.4.y+z is compatible with 1.0.0, if you land a breaking change bump
 # the version to 2.0.0.
 # See more details: https://github.com/flutter/flutter/wiki/Package-migration-to-1.0.0
-version: 0.4.9+2
+version: 0.4.10
```

Comment:
As above, I think this should be at least 0.5 given the significance of the change.

Reply:
updated to 0.5.0